tests/test_models.py

```diff
@@ -0,0 +1,16 @@
+def test_new_user(client):
+    """
+    GIVEN a User model
+    WHEN a new User is created
+    THEN check if the username and password fields are defined correctly
```

Comment:
Is this `GIVEN` `WHEN` `THEN` doctoring a special syntax for pytest?

Reply:
No, this is just a docstring. No idea why @hmoldenhauer put it that way